### DISCUSSION TITLE
VRM Converter導入するあとメニュー出せない

### DISCUSSION THREAD

**Discussion by RainbowO3** [esperecyan/VRMConverterForVRChat#35] [Q&A] [answered]
再起動、パソコン再起動でもメニュー出せない。
![635495389367802496](https://user-images.githubusercontent.com/35169433/159153931-d1a2ed82-37b1-405e-8ed9-162cd4bffbe3.jpg)

**Comment by RainbowO3** [comment]
もう一回導入すると：All assets already been in your project

**Comment by esperecyan** [comment, marked as answer]
新規Unityプロジェクトで、公式ページのインストーラーをご利用ください。
https://pokemori.booth.pm/items/1025226
